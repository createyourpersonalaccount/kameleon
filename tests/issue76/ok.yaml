global:
    arch: x86_64
    iso_arch: $${arch}
    iso_filename: debian-jessie-$${iso_arch}-live.iso 

bootstrap:
    - testbootstrap:
      - step:
        - exec_local: echo $iso_filename
