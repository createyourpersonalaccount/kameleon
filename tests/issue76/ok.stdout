Starting recipe consistency check
Resolving variables
Creating kameleon build directory : /home/neyron/scm/OAR/kameleon/tests/issue76/build/ok
Starting build recipe 'ok.yaml'
Step _init_bootstrap took: 0 secs
Step 1 : bootstrap/testbootstrap/step
--> Running the step...
Starting command: "bash"
[local] The local_context has been initialized
[local] 
Step testbootstrap took: 0 secs
Step _clean_bootstrap took: 0 secs
Step _init_setup took: 0 secs
Step _clean_setup took: 0 secs
Step _init_export took: 0 secs
Step _clean_export took: 0 secs

Successfully built 'ok.yaml'
Total duration : 0 secs
